src/com/SocketCommunication.cpp

```diff
@@ -354,7 +354,7 @@ void SocketCommunication::send(std::string const &itemToSend, Rank rankReceiver)
     asio::write(*_sockets[rankReceiver], asio::buffer(&size, sizeof(size_t)));
     asio::write(*_sockets[rankReceiver], asio::buffer(itemToSend.c_str(), size));
   } catch (std::exception &e) {
-    PRECICE_ERROR("Send using sockets failed with system error: {}", e.what());
+    PRECICE_ERROR("Send using sockets failed with system error: {}. This often means that another participant exited with an error (look there).", e.what());
```

Comment:
Actually, I also wanted to change the first sentence, as it confused me for quite a while. I would
1. try to clarify that 'send' refers to an event. Maybe the "send data event" or "while trying to send data"
2. point out that this "send event" refers to inter-participant communication, so that the following hint makes more sense
3. try to clarify that sockets refer to the m2n tag in our config. Many people just use it as a default and never touch it at all so that it's not clear where the sockets belong to. I'm not even sure that the information of communication configuration matters here, as we usually don't want to encourage people to change it when hitting the error message.

Reply:
Very good points, I updated the error messages to reflect all these.